docs/source/blog/Improve-Audio-Transcriptions-with-Label-Studio.md

```diff
@@ -0,0 +1,287 @@
+---
+title: Improve Amazon Transcribe Audio Transcriptions with Label Studio
+type: blog
+order: 95
+meta_title: Improve Audio Transcriptions with Label Studio
+meta_description: Use open source data labeling software Label Studio to improve audio transcriptions of customer support calls, video conference meetings, and other audio recordings.
+---
+
+Audio transcription quality is important for accessibility, but also to ensure the quality of a service that you provide, such as customer support, or the outcomes of a meeting. You might need to transcribe any of the following types of audio:
+- A recording from a journalism, user research, or radio interview
+- Customer support call recordings
+- Depositions for legal cases
+- Business meeting recordings
+- Arbitration discussions
+
+Quality is important, but speed matters too, especially when you need to transcribe a high volume of recordings in a short time frame. 
+
+## Why audio transcription quality matters
+
+For many cases where you're using audio transcriptions, they must be completely accurate so that patterns that you search for in the transcribed content can be easily discovered for research purposes, to help you build a stronger legal case, to more easily improve your product, or to ensure high quality customer support interactions. 
```

Comment:
also worth mentioning: "to reduce error in automated downstream analysis by ML models (like sentiment analysis)